package.json

```diff
@@ -123,12 +123,15 @@
         "i18next-browser-languagedetector": "7.0.1",
         "jotai": "1.11.2",
         "js-file-download": "0.4.12",
+        "jsep": "^1.3.8",
+        "jsonpath-plus": "^7.2.0",
```

Comment:
Please remove `^` from newly added deps. If we don't, local environments will be using different versions and it causes headaches trying to troubleshoot local issues.